### PR TITLE
Modernize async/await code in runtime.js

### PR DESCRIPTION
**Pull request by ChrisAntaki**

### CODE REVIEW

File: src/runtime/runtime-test.js [hunk 1]
```diff
@@ -401,7 +401,7 @@ describes.realWin('Runtime', {}, (env) => {
       resolveStub = sandbox
         .stub(PageConfigResolver.prototype, 'resolveConfig')
         .callsFake(() => configPromise);
-      runtime.configuredPromise_.then((configuredRuntime) => {
+      runtime.configuredRuntimePromise_.then((configuredRuntime) => {
```
Comment (jpettitt) [inline src/runtime/runtime-test.js:404]: Missed a `then()`

**Comment by jpettitt** [comment]
Thank you for commenting the flow

File: src/runtime/runtime-test.js (hunk 1 above)
Comment (ChrisAntaki) [reply src/runtime/runtime-test.js:404]: Nice catch. Looks like the `analyticsMock` expectations weren't being verified either. Looking into fixing this...

Comment (ChrisAntaki) [reply src/runtime/runtime-test.js]: Fixed 👍

**Review by jpettitt — APPROVED** [review]

**Pull request merged**